### PR TITLE
[boards] Add debug capability to Silabs TB2

### PR DESCRIPTION
**Pull request by ekarlso**

### CODE REVIEW

File: boards/arm/efr32mg_sltb004a/board.cmake [hunk 1]
```diff
@@ -0,0 +1,2 @@
+board_runner_args(jlink "--device=EFM32PG12BxxxF1024")
```
Comment (chrta) [inline boards/arm/efr32mg_sltb004a/board.cmake]: This is the wrong device

Comment (ekarlso) [reply boards/arm/efr32mg_sltb004a/board.cmake]: > This is the wrong device
Fuxed ;(

**Review by galak — CHANGES_REQUESTED** [review]
Can you make the commit message something like:

'boards: efr32mg_sltb004a: ...'

We don't tend to use '[ ]' in commit message headers.

**Comment by ekarlso** [comment]
Done

**Comment by galak** [comment]
@chrta look good to you?

**Comment by chrta** [comment]
Yes, looks good.

**Review by chrta — APPROVED** [review]

**Review by galak — APPROVED** [review]

File: boards/arm/efr32mg_sltb004a/board.cmake [hunk 2]
```diff
@@ -0,0 +1,2 @@
+board_runner_args(jlink "--device=EFM32MG12BxxxF1024")
```
Comment (MaureenHelm) [inline boards/arm/efr32mg_sltb004a/board.cmake]: Please add a license/copyright header

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

**Review by MaureenHelm — APPROVED** [review]

File: boards/arm/efr32mg_sltb004a/board.cmake (hunk 2 above)
Comment (MaureenHelm) [reply boards/arm/efr32mg_sltb004a/board.cmake]: cmake comments start with `#`.

https://github.com/zephyrproject-rtos/zephyr/blob/53153405aaaae3d7dd777ba89ca9af69178c218c/boards/arm/mimxrt1050_evk/board.cmake#L1-L5

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**